content/node-placement/docs/eks-node-placement.md

```diff
@@ -111,3 +111,106 @@ Multiple key value pairs for spark.kubernetes.node.selector.[labelKey] can be pa
 `spark.kubernetes.node.selector.[labelKey] - Adds to the node selector of the driver pod and executor pods, with key labelKey and the value as the configuration's value. For example, setting spark.kubernetes.node.selector.identifier to myIdentifier will result in the driver pod and executors having a node selector with key identifier and value myIdentifier. Multiple node selector keys can be added by setting multiple configurations with this prefix.`
 
 
+
+## **Job submitter pod placement**
+
+Similar to driver and executor pods, you can configure the job submitter pod's node selectors as well using the `emr-job-submitter` classification. 
+Using this classification, you can place the job submitter pod in a single AZ or single AZ and ec2 instance type as well.
```

Comment:
Are there properties that are not allowed? If not, can we say this instead: Using this classification, you can place the job submitter pod in a single AZ or using any Kubernetes labels that are applied to the nodes.

Reply:
Addressed comments in latest revision